app/controllers/deployments_controller.rb

```diff
@@ -65,9 +93,9 @@ def repo_path
     end
   end
 
-  def app_name
-    repo_title = repo_path.split("/")[-1].tr("-", " ").humanize.titlecase
-    repo_title.gsub(/\bApi\b/, "API")
+  def app_name(unnormalized_app_name)
```

Comment:
Now that this is a function (which is nice), maybe change the name to be `normalise_app_name` or something like that, which describes what it does.

Reply:
I will do that